sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -0,0 +1,387 @@
+/*
+ * Copyright (C) 2016-17 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
```

Comment:
No doxygen block in this file. I know it is not parsed anyway, but still useful e.g. for the author tags etc and AFAIK it is standard that we put it in every C file, is it not?